Resources/Prototypes/Construction/structures.yml

```diff
@@ -33,3 +33,65 @@
   steps:
   - material: Metal
     amount: 2
+
+- type: construction
+  name: Window
+  id: window
+  category: Structures
+  description: Like a wall, but opaque!
```

Comment:
but... windows aren't opaque... and walls are!!!

Reply:
TIL i've been using opaque wrong